core/ui/src/main/kotlin/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCard.kt

```diff
@@ -112,7 +128,22 @@ fun NewsResourceCardExpanded(
                     Row {
                         NewsResourceTitle(
                             userNewsResource.title,
-                            modifier = Modifier.fillMaxWidth((.8f)),
+                            modifier = Modifier.fillMaxWidth((.8f))
+                                .dragAndDropSource {
```

Comment:
I wonder if the whole card should be draggable, instead of just the title?

Reply:
I'd rather avoid using the whole card, as we have no clear indication of what is draggable and what is not, and it might confuse users when they tap longer on the card